scss/grid/_layout.scss

```diff
@@ -254,8 +254,6 @@
 
 
         .k-header {
-            height: button-size();
-            line-height: button-size();
```

Comment:
This line-height is needed. Please check the following issue: 
https://github.com/telerik/kendo-theme-default/issues/284